rollup.config.js

```diff
@@ -0,0 +1,62 @@
+var nodeResolve = require('rollup-plugin-node-resolve');
+var commonjs = require('rollup-plugin-commonjs');
+var babel = require('rollup-plugin-babel');
+var uglify = require('rollup-plugin-uglify');
+
+var env = function( name, def ){
+  var val = process.env[ name ];
+
+  if( val === undefined || val === '' ){
+    return def;
+  } else {
+    return val;
+  }
+};
+var boolEnv = function( name, def ){
+  return env( name, def ? 'true' : 'false' ) == 'true';
+};
+var FILENAME = env('FILENAME', 'cytoscape.js');
+var NODE_ENV = env('NODE_ENV', '');
+var MINIFY = boolEnv('MINIFY', false);
+var BABEL = boolEnv('BABEL', true);
+var SOURCEMAPS = boolEnv('SOURCEMAPS', false);
+var pkg = require('./package.json');
+var path = require('path');
+
+module.exports = {
+  input: './src/index.js',
+  output: {
+    file: path.resolve(__dirname, 'build', FILENAME),
+    name: 'cytoscape',
+    format: NODE_ENV === 'production' ? 'cjs' : 'umd',
```

Comment:
Probably always need umd

Reply:
There is already umd here with bundled dependencies. This version is clean node module. Also umd will require global aliasing for missing deps.

Reply:
The current UMD made by Webpack allows for using external dependencies with each of globals, cjs, amd.  Is this not possible with Rollup?  

Would Rollup be able to automatically support cjs and globals with external dependencies?  If we go this route (dropping amd support), we'd have to release a new major version (v4).